besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1218,16 +1218,6 @@ static class TxPoolOptionGroup {
             "Price bump percentage to replace an already existing transaction  (default: ${DEFAULT-VALUE})",
         arity = "1")
     private final Integer priceBump = TransactionPoolConfiguration.DEFAULT_PRICE_BUMP.getValue();
-
-    @Option(
-        names = {"--tx-pool-future-max-by-account"},
-        paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
-        converter = PercentageConverter.class,
-        description =
-            "Maximum per account of currently unexecutable future transactions that can occupy the txpool (default: ${DEFAULT-VALUE})",
-        arity = "1")
-    private final Integer maxFutureTransactionsByAccount =
-        TransactionPoolConfiguration.MAX_FUTURE_TRANSACTION_BY_ACCOUNT;
```

Comment:
I think we need to keep this for the moment, otherwise it is a breaking change, what we can do is to give an error if this and the new one are used together, or to say that this has no effect if the new one is used